packages/components/package.json

```diff
@@ -25,40 +26,27 @@
     "lint:typescript": "../../scripts/check-typescript"
   },
   "devDependencies": {
-    "@babel/runtime": "^7.12.5",
-    "date-fns": "^2.16.1",
     "nebenan-eventproxy": "^1.0.0",
     "nebenan-form": "^10.0.0",
-    "nebenan-helpers": "^7.1.0",
     "nebenan-keymanager": "^1.0.1",
-    "nebenan-ui-kit": "^5.1.0",
     "node-sass-glob-importer": "^5.3.2",
-    "prop-types": "^15.7.2",
-    "qs": "^6.9.4",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
-    "react-load-script": "^0.0.6",
     "react-redux": "^7.2.2",
-    "react-router": "^5.2.0",
-    "react-router-dom": "^5.2.0",
-    "redux": "^4.0.5",
-    "sass-functions": "^2.0.0",
-    "smartcontent": "^3.10.0",
-    "string-validate": "^3.5.0"
+    "redux": "^4.0.5"
   },
   "dependencies": {
-    "@goodhood/icons": "^3.0.0",
-    "clsx": "^1.1.1",
-    "lodash": "^4.17.20",
-    "@popperjs/core": "^2.5.4",
+    "@goodhood/icons": "^3.5.0",
+    "@popperjs/core": "^2.11.6",
+    "clsx": "^1.2.1",
+    "lodash": "^4.17.21",
     "escape-html": "^1.0.3",
-    "marked": "^4.0.0",
-    "nebenan-react-hocs": "^9.0.0"
+    "marked": "^4.2.5",
+    "nebenan-react-hocs": "^9.0.0",
+    "react-load-script": "^0.0.6"
```

Comment:
this needs to stay a peer dep (it has a global register of scripts). maybe you accidentally added it here instead of devDependencies?

Reply:
Still stays as a peer dep, i just moved it into deps from devdeps since it is used in advertimesnt component. anyway, will undo the package updates to get this PR forward